modul/tutorial.py

```
@@ -160,7 +172,13 @@ def update(self, dt, events):
 
         for event in events:
             if event.type == pygame.KEYDOWN:
-                if event.key == pygame.K_ESCAPE or event.key == pygame.K_SPACE:
+                from modul import input_utils
+                pause_key = input_utils.get_action_keycode("pause")
+                shoot_key = input_utils.get_action_keycode("shoot")
+                if ((pause_key is not None and event.key == pause_key)
+                        or event.key == pygame.K_ESCAPE
+                        or (shoot_key is not None and event.key == shoot_key)
+                        or event.key == pygame.K_SPACE):
```

Pylint (4.0.4)

[refactor] Too many boolean expressions in if statement (6/5)

(R0916)